README.md

```diff
@@ -98,6 +98,7 @@ In case you are using the browser version (directly including the script on your
 VueScrollTo.setDefaults({
     container: "body",
     duration: 500,
+    dynamicPosition: true,
```

Comment:
I'm not sure about the naming, perhaps `lazy: true` (the default) and `false` to recalculate for every step.

Reply:
Okay, no problem